packages/components/src/core/Callout/index.tsx

```diff
@@ -56,7 +56,7 @@ const Callout = ({
   };
 
   const getIcon = () => {
-    if (icon) return icon;
+    if (icon !== undefined) return icon;
```

Comment:
Niice! Thanks so much for the fix, @liaprins 🙏 ❤️ 

Can we add a comment to explain why we explicitly use `icon !== undefined` here please? In case someone in the future "refactors" this back to `if (icon) return icon` lol

Something like:

```ts
/**
 * (liaprins): Explicitly use `icon !== undefined` to fix bug in ticket: https://github.com/chanzuckerberg/sci-components/issues/577 
 */
```

Or something better!

Reply:
Great, thanks for the suggestion! I have finally updated to add the comment as suggested!